src/fstests/runner.py

```diff
@@ -67,60 +67,42 @@ def _run_single_job(self, tarball_node, device):
         finally:
             return True
 
-    def check_kvm_xfstest_env(self):
-        try:
-            result = subprocess.run(
-                "kvm-xfstests --help", check=True, shell=True)
-            if result.returncode == 0:
-                print(f"KVM-xftests found.")
-                return True
-            else:
-                return False
-        except Exception as e:
-            print(f"Raised exception: {e}")
+    def run(self, args):
+        if args.node_id:
+            tarball_node = self._db.get_node(args.node_id)
+        else:
+            tarball_node = None
```

Comment:
This is where you can listen for an event from the pub/sub API to get a new tarball node rather than providing the id by hand.